src/common/components/ui/formik/validationSchemas.ts

```diff
@@ -423,3 +424,33 @@ export const bmcAddressValidationSchema = Yup.string().matches(BMC_REGEX, {
   message: 'Value "${value}" is not valid BMC address.', // eslint-disable-line no-template-curly-in-string
   excludeEmptyString: true,
 });
+
+export const LOCATION_VALIDATION_MESSAGES = {
+  INVALID_LENGTH: '1-63 characters',
+  INVALID_VALUE: 'Use alphanumberic characters, dot (.), underscore (_) or hyphen (-)',
+  INVALID_START_END: 'Must start and end with an alphanumeric character',
+};
+
+export const locationValidationSchema = Yup.string()
+  .min(1, NAME_VALIDATION_MESSAGES.INVALID_LENGTH)
+  .max(63, NAME_VALIDATION_MESSAGES.INVALID_LENGTH)
```

Comment:
```suggestion
  .min(1, LOCATION_VALIDATION_MESSAGES.INVALID_LENGTH)
  .max(63, LOCATION_VALIDATION_MESSAGES.INVALID_LENGTH)
```

otherwise looks good :)

Reply:
thanks for your review!